src/navigation/path.cpp

```diff
@@ -243,45 +260,59 @@ glm::dquat Path::easedSlerpRotation(double t) const {
 }
 
 glm::dquat Path::linearPathRotation(double t) const {
-    const double tHalf = 0.5;
-
-    const glm::dvec3 endNodePos = _end.node()->worldPosition();
-    const glm::dvec3 endUp = _end.rotation() * glm::dvec3(0.0, 1.0, 0.0);
-
-    if (t < tHalf) {
-        // Interpolate to look at target
-        const glm::dvec3 halfWayPosition = _curve->positionAt(tHalf);
-        const glm::dquat q = ghoul::lookAtQuaternion(halfWayPosition, endNodePos, endUp);
-
-        const double tScaled = ghoul::sineEaseInOut(t / tHalf);
-        return glm::slerp(_start.rotation(), q, tScaled);
-    }
-
-    // This distance is guaranteed to be strictly decreasing for linear paths
-    const double distanceToEnd = glm::distance(_prevPose.position, _end.position());
-
-    // Determine the distance at which to start interpolating to the target rotation.
-    // The magic numbers here are just randomly picked constants, set to make the
-    // resulting rotation look ok-ish
-    double closingUpDistance = 10.0 * _end.validBoundingSphere();
-    if (pathLength() < 2.0 * closingUpDistance) {
-        closingUpDistance = 0.2 * pathLength();
-    }
-
-    if (distanceToEnd < closingUpDistance) {
-        // Interpolate to target rotation
-        const double tScaled = ghoul::sineEaseInOut(1.0 - distanceToEnd / closingUpDistance);
-
-        // Compute a position in front of the camera at the end orientation
-        const double inFrontDistance = glm::distance(_end.position(), endNodePos);
-        const glm::dvec3 viewDir = ghoul::viewDirection(_end.rotation());
-        const glm::dvec3 inFrontOfEnd = _end.position() + inFrontDistance * viewDir;
-        const glm::dvec3 lookAtPos = ghoul::interpolateLinear(tScaled, endNodePos, inFrontOfEnd);
-        return ghoul::lookAtQuaternion(_prevPose.position, lookAtPos, endUp);
-    }
-
-    // Keep looking at the end node
-    return ghoul::lookAtQuaternion(_prevPose.position, endNodePos, endUp);
+    const glm::dvec3 a = ghoul::viewDirection(_start.rotation());
+    const glm::dvec3 b = ghoul::viewDirection(_end.rotation());
+    const double angle = std::acos(glm::dot(a, b)); // assumes length 1.0 for a & b
+
+    // Seconds per pi angles. Per default, it takes 5 seconds to turn 90 degrees
+    double factor = 5.0 / (0.5 * glm::pi<double>()); // TODO: make property to adapt this
```

Comment:
`0.5 * glm::pi<double>()` -> `glm::half_pi<double>()`

Reply:
And remove outdated comment